lms/djangoapps/student_account/views.py

```diff
@@ -592,3 +596,45 @@ def account_settings_context(request):
         } for state in auth_states if state.provider.display_for_login or state.has_account]
 
     return context
+
+
+class RecoverPasswordView(ViewSet):
+
+    """
+    **Use Cases**
+            Resets a password of a user an email address
+        **Example Requests**:
+            POST /account/recover-password
+            {"email": "staff@example.com"}
+
+        **Response Values**
+            200 or 400
+    """
+
+    def post(self, request, format=None):
+        data = request.data
+        if 'email' in data:
+            limiter = BadRequestRateLimiter()
+            if limiter.is_rate_limit_exceeded(request):
+                AUDIT_LOG.warning("Password reset rate limit exceeded")
+                return HttpResponseForbidden()
+
+            email = data['email']
+
+            if len(email) > 0:
```

Comment:
Is this check necessary? If it's empty it would fall through the catch block. The exception I suppose is if we initialized a superuser without an email through the command line.